python/GafferSceneUI/SceneInspector.py

```diff
@@ -987,6 +1002,18 @@ def __showHistory( self, target ) :
 		self.ancestor( GafferUI.Window ).addChildWindow( w, removeOnClose = True )
 		w.setVisible( True )
 
+	def __showShaderParameters( self, target ) :
+
+		w = _SectionWindow(
+			target.scene.node().getName() + " : " + self.__label().getText(),
```

Comment:
Since we'll need to refactor _SectionWindow a little anyway to support multiple targets, it probably makes sense for the window constructor to generate the title from the targets. This would also allow it to get the title right even when the node name changes.